source/reference/minio-mc/mc-ilm-tier-add.rst

```diff
@@ -0,0 +1,343 @@
+
+   :mc-cmd:`mc ilm tier add` replaces ``mc admin tier add``.
+
+Description
+-----------
+
+.. start-mc-ilm-tier-add-desc
+
+The :mc:`mc ilm tier add` command creates a new remote storage tier to a supported S3-compatible service for MinIO :ref:`Lifecycle Management: Object Transition ("Tiering")<minio-lifecycle-management-expiration>`. 
+
+.. end-mc-ilm-tier-add-desc
+
+Supported S3 Services
+~~~~~~~~~~~~~~~~~~~~~
+
+:mc:`mc ilm tier add` supports *only* the following S3-compatible services as a remote target for object tiering:
+
+- MinIO
+- Amazon S3
+- Google Cloud Storage
+- Azure Blob Storage
+
+Permissions
+~~~~~~~~~~~
+
+MinIO requires the following administrative permissions on the cluster in which you create remote tiers for object transition lifecycle management rules:
+
+- :policy-action:`admin:SetTier`
+- :policy-action:`admin:ListTier`
+
+For example, the following policy provides permission for configuring object transition lifecycle management rules on any bucket in the cluster: 
+
+.. literalinclude:: /extra/examples/LifecycleManagementAdmin.json
+   :language: json
+   :class: copyable
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following example creates a new remote tier called ``WARM-MINIO-TIER`` on the ``myminio`` deployment.
+      The command creates a tier for a remote MinIO deployment located at the hostname ``https://warm-minio.com``.
+      
+      .. code-block:: shell
+         :class: copyable
+
+          mc ilm tier add minio myminio WARM-MINIO-TIER                     \
+                                        --endpoint https://warm-minio.com   \                       
+                                        --access-key ACCESSKEY              \
+                                        --secret-key SECRETKEY              \
+                                        --bucket mybucket                   \
+                                        --prefix myprefix/ 
+
+      Lifecycle management rules on the ``myminio`` deployment can use the new tier to transition objects into the remote location's ``myprefix/`` prefix in the ``mybucket`` bucket.
+
+   .. tab-item:: SYNTAX
+   
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc ilm tier add TIER_TYPE                    \
+                         TARGET                       \
+                         TIER_NAME                    \
+                         --endpoint string            \
+                         [--region string]            \
+                         [--access-key value]         \
+                         [--secret-key value]         \
+                         [--use-aws-role]             \
+                         [--account-name value]       \
+                         [--account-key value]        \
+                         [--credentials-file value]   \
+                         --bucket value               \
```

Comment:
Move this up before `--endpoint` since it's required

Reply:
👍